packages/client/src/index.test.ts

```diff
@@ -113,7 +112,7 @@ describe('LogtoClient', () => {
     test('claims restored', async () => {
       const storage = new MemoryStorage();
       storage.setItem(
-        `LOGTO_TOKEN_SET_CACHE::${discoverResponse.issuer}::${CLIENT_ID}::${DEFAULT_SCOPE}`,
+        `LOGTO_TOKEN_SET_CACHE::${discoverResponse.issuer}::${CLIENT_ID}::${DEFAULT_SCOPE_STRING}`,
```

Comment:
the string itself should be a const as well?

Reply:
- All string constants:
    The constants looks like symbol links.
    When we need to modify its value, we can modify its value at the correct position (where symbol link referenced to) and then rename it.
    It's more accurate than replacing all strings via searching because the variables with the same values maybe have the different meanings.
- `DEFAULT_SCOPE_STRING`:
    So far it contains 'openid' and 'offline_access'.
    Maybe it will contain more scope values.
    Extract it to a constant so that we can make it easier to modify.